src/coreclr/src/pal/src/misc/perftrace.cpp

```diff
@@ -1,1344 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-/*++
-
-
-
-Module Name:
-
-    misc/perftrace.c
-
-Abstract:
-    Implementation of PAL Performance trace utilities.
-
-
-
---*/
-
-/* PAL headers */
-
-
-
-#ifdef PAL_PERF
```

Comment:
If you delete this file, `PAL_PERF` will become useless. If you believe that it is not useful, it may be best to cleanup the whole thing in separate PR.

Reply:
Sounds good, will exclude it from this PR. This file was found because is not included in the build. Looks like `PAL_PERF` was never defined and `PERFInitialize` is unused in pal.cpp since the initial port.